src/utils/download.ts

```diff
@@ -0,0 +1,54 @@
+import * as fs from 'fs';
+import path from 'path';
+import axios from 'axios';
+
+type DownloadProgressCallback = (downloadProgress: number) => void;
+type DownloadResult = { pathToFile: string; contentType: string };
+
+/**
+ * Downloads file from remote HTTP[S] host and puts its contents to the
+ * specified location.
+ * @param url URL of the file to download.
+ * @param destinationPath Path to the destination file.
+ * @param destinationFileName The file name.
+ */
+
+export async function download(
```

Comment:
Similar to the `temp` comment, `download` is too generic. It is clear here that we are only using this to download files. I would expect both the function and file to have a more descriptive name